### DISCUSSION TITLE
Allow negative values in cloud input for different LightingInfos (has been proven useful).

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4688] [Ideas]
Someone (user Finis) came to me today annoyed because they didn't want clouds at night. So I went ahead and changed `LevelLighting.times[3].singles[2]` (cloud float) to `-10` and it worked. The game properly clears the clouds prior to midnight and slowly brings them back come daytime. These changes also apply when saved.

So there are two options to implement this:
- Have a toggle to turn off clouds for a specific time/LightingInfo. You don't need to store a new bool, you can just set a negative value like I did.
- Simply add a value input next to the slider so that we can go into the negative values.

Either way, this is relatively easy to implement and would pair well with maps that have the aroura borealis (northern lights). The original person that wanted this said they wanted players to enjoy the night sky.

**Comment by CPL-M** [comment]
I can see a lot of themes where this would be useful and a few other ideas that need this to work properly

**Comment by SDGNelson** [comment]
Hmm interesting workaround. 👀

**Comment by Jdance-Media** [comment]
Recently a user in the Unturned discord made a post about wanting this feature: https://discord.com/channels/324229387295653889/1384416184367190077/1384416184367190077

I don't think it is something that needs to be prioritized at all, but I believe adding an actual value box next to the cloud slider would be relatively easy to do and would be well paired with an upcoming quality of life update (e.g. the new road system in the editor).

**Comment by CPL-M** [comment]
There is actually a way to do this via level assets, but I feel like it's way too out of the way for something like disabling clouds

**Reply by Jdance-Media** [reply]
I agree, I feel like most aspects of making maps should be kept UI based and user friendly. It seems that understanding LevelAssets is a challenge for many whom haven't touched .dats or .assets before.

However, I'd like to make it clear that the LevelAsset option does not allow for us to disable clouds for certain times of the day. Not to mention a more elegant way to do it built in with an already existing value it might still be worth exploring.
